tests/datamodules/test_levircd.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+from pathlib import Path
+
+import pytest
+from lightning.pytorch import Trainer
+from pytest import MonkeyPatch
+
+import torchgeo.datasets.utils
+from torchgeo.datamodules import LEVIRCDPlusDataModule
+from torchgeo.datasets import LEVIRCDPlus
+
+
+def download_url(url: str, root: str, *args: str) -> None:
+    shutil.copy(url, root)
+
+
+class TestLEVIRCDPlusDataModule:
+    @pytest.fixture(params=["train", "validate", "test"])
```

Comment:
You don't seem to be using this parameterization anywhere. You're running all tests 3 times with the exact same inputs.